hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/ReadableStakingInfoStoreImpl.java

```diff
@@ -39,16 +50,17 @@ public StakingNodeInfo get(final long nodeId) {
     @NonNull
     @Override
     public Set<Long> getAll() {
-        // For entity types that have many instances this code would be a bad idea, but for node staking info there
-        // should only be a limited number of staking nodes in state. Iterating over all of them should not be expensive
-        final var keysIter = stakingInfoState.keys();
-        if (!keysIter.hasNext()) return Collections.emptySet();
-
+        final var numStakingInfo = entityCounters.getCounterFor(EntityType.STAKING_INFO);
+        if (numStakingInfo == 0) {
+            return Collections.emptySet();
+        }
         final var nodeIds = new HashSet<Long>();
-        while (keysIter.hasNext()) {
-            nodeIds.add(keysIter.next().number());
+        for (var i = 0; i < numStakingInfo; i++) {
+            final var nodeId = EntityNumber.newBuilder().number(i).build();
```

Comment:
Why some places use `new EntityNumber(i)`, while others use a builer?

Reply:
Yes, agreed inconsistency. Will fix to using `new EntityNumber(i)`. Thanks